### PR TITLE
Added jupyter notebook references english

### PR DESCRIPTION
**Pull request by rakshith6404**
PR-Hactoberfest2022
I have added Jupyter-english reference regarding #8121 
Looking forward to merge it :smile:

### CODE REVIEW

File: more/free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -222,6 +223,12 @@
 * [Try jQuery](http://try.jquery.com)
 
 
+### Jupyter
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-en.md:226]: these two resources should be in books, as a new category under python

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by rakshith6404** [comment]
I have made changes for the following please look into it.
I am looking forward to merge this file

File: more/free-programming-interactive-tutorials-en.md (hunk 1 above)
Comment (eshellman) [reply more/free-programming-interactive-tutorials-en.md]: move them to the books list

**Comment by rakshith6404** [comment]
Hello Have added/Updated the requested changes yesterday 
i have added jupyter under python as a category , added related youtube tutorial(playlist) , added jupyter cheatsheet and also added jupyter youtube playlist in hindi .
https://github.com/rakshith6404/free-programming-books/blob/added_jupyter_notebook_references_english/more/free-programming-interactive-tutorials-en.md
https://github.com/rakshith6404/free-programming-books/blob/added_jupyter_notebook_references_english/courses/free-courses-hi.md
Looking forward for a review !!!! :smile:

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**